presto-product-tests/src/main/java/com/facebook/presto/tests/cassandra/TestSelect.java

```diff
@@ -179,13 +180,13 @@ public void testAllDataTypes()
                 .containsOnly(
                         row("\0", Long.MIN_VALUE, Bytes.fromHexString("0x00").array(), false, 0f, Double.MIN_VALUE,
                                 Float.MIN_VALUE, "[0]", "0.0.0.0", Integer.MIN_VALUE, "[0]", "{\"\\u0000\":-2147483648,\"a\":0}",
-                                "[0]", "\0", parseTimestampInLocalTime("1970-01-01 00:00:00.0", PRODUCT_TESTS_TIME_ZONE),
```

Comment:
so the `PRODUCT_TESTS_TIME_ZONE` was completely ignored here?

Reply:
No, it wasn't. It had to match JVM time zone though (because of how `java.sql.Timestamp` internal repr works), and now this is done automatically via `Timestamp.valueOf(LocalDateTime)`

Reply:
> No, it wasn't. It had to match JVM time zone though (because of how java.sql.Timestamp internal repr works)

Looking at the:
```
    public static Timestamp parseTimestampInLocalTime(String value, DateTimeZone timeZone)
    {
        return new Timestamp(DATE_TIME_FORMATTER.withZone(timeZone).parseMillis(value));
    }
```
it used `parseMillis(value)`. And `parseMillis(value)` should be independent of `timeZone` parameter, no?

Reply:
Specifically `parseMillis` states:
```
     * The parse will use the ISO chronology, and the default time zone.
     * If the text contains a time zone string then that will be taken into account.
```

Reply:
the result of `DATE_TIME_FORMATTER.withZone(timeZone).parseMillis(value)` depends on `timeZone` when `value` doesn't specify time zone.